themes/beaver/layouts/blog/list.html

```diff
@@ -6,63 +6,47 @@
-                  </div>
-                  <br />
-                  <br />
-                  <div class="fl-module fl-module-pp-content-grid fl-node-o6rwlh2bvzie" data-node="o6rwlh2bvzie">
-                    <div class="fl-module-content fl-node-content">
-
-                      {{ range (.Paginate (.Pages.ByParam "created_at").Reverse).Pages }}
-                      <a class="link" href="{{ .Permalink }}">
-                        <div class="blog-post">
-                          <div class="post-image">
-                            {{ $cdnPrefix := "https://imagecdn.app/v1/images/" }}
-                            {{ $src := "" }}
-                            {{ with .Params.cover_image }}
-                              {{ $encodedSrc := printf "%s%s" $cdnPrefix (urlquery .) }}
-                              {{ $src = printf "%s?width=150&height=150" $encodedSrc }}
-                            {{ else }}
-                              {{ $src = "/assets/img/no-image.png" }}
-                            {{ end }}
+            <div style="max-width: 960px; margin:0 auto; padding-left:18px;padding-right:18px">
+              <div style="max-width: 43rem;">
+                <h1 class="fl-heading" style="margin: 0!important;padding: 0!important">Blog</h1>
 
-                            <img src="{{ $src }}" alt="Placeholder Image" loading="lazy" decoding="async">
+                <p>Browse through our blog page filled with updated information and the
+                latest tips to help you achieve your goals.</p>
 
-                          </div>
-                          <div class="post-content">
-                            <h2 class="post-title">{{ .Title }}</h2>
-                            <div class="post-meta">
-                              <div class="post-tags">{{ range .Params.tags }}<a href="/blog/tags/{{ . | urlize }}/">#{{ . }}</a> {{ end }}</div>
-                              {{ if .Params.created_at }}
-                              <span class="post-date">{{ dateFormat "Jan 2, 2006" .Params.created_at }}</span>
-                              {{ end }}
-                            </div>
-                          </div>
-                        </div>
-                      </a>
+                <br>
+              
+                {{ range (.Paginate (.Pages.ByParam "created_at").Reverse).Pages }}
+                <a class="link" href="{{ .Permalink }}">
+                  <div class="blog-post">
+                    <div class="post-image">
+                      {{ $cdnPrefix := "https://imagecdn.app/v1/images/" }}
+                      {{ $src := "" }}
+                      {{ with .Params.cover_image }}
+                        {{ $encodedSrc := printf "%s%s" $cdnPrefix (urlquery .) }}
+                        {{ $src = printf "%s?width=150&height=150" $encodedSrc }}
+                      {{ else }}
+                        {{ $src = "/assets/img/no-image.png" }}
                       {{ end }}
```

Comment:
@dgorodnichy please schedule to cleanup this code: move to shortcode I believe would be enough